azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java

```diff
@@ -688,17 +688,22 @@ private void ajaxFetchJobInfo(final Project project, final HashMap<String, Objec
       return;
     }
 
-    final Props prop;
+    Props prop;
     try {
-      prop = this.projectManager.getProperties(project, node.getJobSource());
+      prop = this.projectManager.getProperties(project, flow, jobName, node.getJobSource());
```

Comment:
Is this job Prop? Can we detail the name as jobProp?

Reply:
Yes, here it's job prop.